### ISSUE TITLE
Password Manage

### ISSUE THREAD

**Issue by lawson85** [scoute-dich/browser#293]
It would be good if the browser offered to save passwords. I'm a new user

**Comment by scoute-dich** [comment]
Most websites will save your login data, if you enable JavaScript and cookies for the site. If not, you can save it as a bookmark. If you edit this bookmark you can save your login data. When you open the bookmark you'll get a notification to copy your login data to the clipboard.

**Comment by scoute-dich** [comment]
There's no native way in Android webview to explicit save passwords. This feature has been removed by Google a few Android versions ago.

**Comment by Ascrod** [comment]
I'm speculating a bit, but I believe autofill was removed from WebView in favor of using Android's autofill service. Android by default uses Google for this, but you can change it to LastPass, KeePass, or something else.

**Comment by puppykickr** [comment]
I found that the autofill wil not work correctly on many browsers like this one.

I was using Bitwarden for a while, but the autofill would not search for the proper login.

It was always searching for the apk name of the browser I was using.

After lengthy discussion with a representative of Bitwarden I was told that the code required for autofill to work correctly has to be implemented by the developer of the browser.

Also stated was the fact that Bitwarden itself was only made to be compatible with the most popular browsers.

I mentioned the fact that since Bitwarden is now available on F-Droid, it might be of interest to them to make their app compatible with the browsers available on that platform.

As it is, I have tried LastPass, KeePass, and multiple others-  only to have them not work properly as well.

I believe that it is due to the same issues with Bitwarden.

So I have given up on the use of any password management app, to avoid the frustration.

There is no way that I am going to use Chrome or FireFox just so a password manager app has capability.

That defeats the whole purpose of using superior browsers like this one.
